internal/librarian/release_init.go

```diff
@@ -37,10 +37,33 @@ const (
 // cmdInit is the command for the `release init` subcommand.
 var cmdInit = &cli.Command{
 	Short:     "init initiates a release by creating a release pull request.",
-	UsageLine: "librarian release init [arguments]",
-	Long: `The release init command is the primary entry point for initiating a release.
-It orchestrates the process of parsing commits, determining new versions, generating
-a changelog, and creating a release pull request.`,
+	UsageLine: "librarian release init [flags]",
+	Long: `The 'release init' command is the primary entry point for initiating
+a new release. It automates the creation of a release pull request by parsing
+conventional commits, determining the next semantic version for each library,
+and generating a changelog. Librarian is environment aware and will check if the
+current directory is the root of a librarian repository. If you are not
+executing in such a directory the '--repo' flag must be provided.
+
+This command scans the git history since the last release, identifies changes
+(feat, fix, BREAKING CHANGE), and calculates the appropriate version bump
+according to semver rules. It then delegates all language-specific file
+modifications, such as updating a CHANGELOG.md or bumping the version in a pom.xml, 
+to the configured language-specific container.
+
+By default, 'release init' leaves the changes in your local working directory
+for inspection. Use the '--push' flag to automatically commit the changes to
```

Comment:
I think we should also mention `--commit` flag here.

Reply:
done